app/backend/src/couchers/models/User.py

```diff
@@ -0,0 +1,219 @@
+import enum
+from calendar import monthrange
+from datetime import date
+
+from geoalchemy2.types import Geometry
+from sqlalchemy import BigInteger, Boolean, Column, Date, DateTime, Enum, Float, Integer
+from sqlalchemy import LargeBinary as Binary
+from sqlalchemy import MetaData, String
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.ext.hybrid import hybrid_property
+from sqlalchemy.orm.session import Session
+from sqlalchemy.sql import func
+
+from couchers.config import config
+from couchers.utils import get_coordinates
+
+meta = MetaData(
+    naming_convention={
+        "ix": "ix_%(column_0_label)s",
+        "uq": "uq_%(table_name)s_%(column_0_name)s",
+        "ck": "ck_%(table_name)s_%(constraint_name)s",
+        "fk": "fk_%(table_name)s_%(column_0_name)s_%(referred_table_name)s",
+        "pk": "pk_%(table_name)s",
+    }
+)
+
+Base = declarative_base(metadata=meta)
```

Comment:
These need to be moved into `__init__.py`, I think

Reply:
Or to something like `base.py`

Reply:
I moved it to base.py